### DISCUSSION TITLE
Welcome to fluffy-computing-machine Discussions!

### DISCUSSION THREAD

**Discussion by cooldude94-R** [cooldude94-R/fluffy-computing-machine#5] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by pradosh-arduino** [comment]
Cool!

**Comment by pradosh-arduino** [comment]
hi!
